os/mm/mm_heap/mm_heapinfo_parse_heap.c

```diff
@@ -64,6 +64,12 @@
 #define HEAPINFO_INT INT16_MAX
 #define HEAPINFO_NONSCHED (INT16_MAX - 1)
 
+#if defined(CONFIG_MM_ASSERT_ON_FAIL) && defined (__KERNEL__)
+#define mfdbg lldbg
+#else  /* CONFIG_MM_ASSERT_ON_FAIL */
+#define mfdbg printf
+#endif
```

Comment:
I don't know why this is necessary. Could you let me know the reason why this is added?